addons/product/views/product_template_views.xml

```diff
@@ -95,8 +95,8 @@
                 <field name="categ_id"/>
                 <progressbar field="activity_state" colors='{"planned": "success", "today": "warning", "overdue": "danger"}'/>
                 <templates>
-                    <t t-name="kanban-card" class="row g-0">
-                        <main class="col-10 pe-2">
+                    <t t-name="kanban-card" class="o_product_kanban_card flex-row">
+                        <main class="flex-grow-1 w-100 pe-2">
```

Comment:
I don't see a scenario in which `<main>` shouldn't grow, IMO we can add `flex-grow: 1;`  in the CSS directly and remove the class form here.